docs/dashboards/navigator-component.md

```diff
@@ -5,8 +5,8 @@ Navigator Component
 
 ## Overview
 
-The NavigatorComponent is part of the Dashboards plugins. It utilizes the
-navigator of Highcharts Stock to provide an overview over the values of a table
+The `NavigatorComponent` is part of the `Dashboards` plugins. It utilizes the
+navigator of `Highcharts Stock` to provide an overview over the values of a table
```

Comment:
I'm not sure if for **Dashboards** and **Highcharts Stock** we should use \` here. For `Navigate Component`, yes, because it is part of the code. If we want to highlight, can we write them in italics or bold? By the way, for Highcharts Stock you can add a URL to an article with the navigator.

Reply:
I agree with Dawid, especially that in other parts of the docs we use `**Dashboard**`

Reply:
I agree, these are brand names and it will be enough to capitalize them. I will fix it.